test/unit/scval_test.js

```diff
@@ -182,12 +182,12 @@ describe('parsing and building ScVals', function () {
     scv = nativeToScVal(
       {
         hello: 'world',
-        goodbye: [1, 2, 3]
+        there: [1, 2, 3]
```

Comment:
it's tough to grasp how expectations are validating the new sorting of keys from this test.

Reply:
So the main test for order was the earlier one. This test was evaluating that `type` was being used correctly, and there are some assertions about `scv[0]` and `scv[1]` that become out of order since `goodbye` comes before `hello` lexicographically, so instead of modifying the assertion checks I just changed the key name so it'd stay in the same order as before.